modules/olm-operatorgroups-copied-csvs.adoc

```diff
@@ -10,3 +10,8 @@ OLM creates copies of all active member CSVs of an Operator group in each of the
 Copied CSVs have a status reason `Copied` and are updated to match the status of their source CSV. The `olm.targetNamespaces` annotation is stripped from copied CSVs before they are created on the cluster. Omitting the target namespace selection avoids the duplication of target namespaces between tenants.
 
 Copied CSVs are deleted when their source CSV no longer exists or the Operator group that their source CSV belongs to no longer targets the namespace of the copied CSV.
+
+[NOTE]
+====
+The `disableCopiedCSVs` toggle must be `false`. After disabling a `disableCopiedCSVs` toggle, OLM deletes existing copied CSVs on a cluster. When a `disableCopiedCSVs` toggle is enabled, OLM adds Copied CSVs again.
```

Comment:
- Suggestion>> since disabled/enabled is used, do we want to modify the first sentence to ` The `disableCopiedCSVs` toggle must be disabled.`
-  Would suggest adding an article before OLM >>`the OLM  deletes....` .
- When a `disableCopiedCSVs` toggle is enabled, OLM adds Copied CSVs again. >>  When a `disableCopiedCSVs` toggle is enabled, the OLM adds the copied CSVs again.

Reply:
Addressed